where_my_money/transactions.py

```diff
@@ -0,0 +1,245 @@
+"""Classes for transaction mapping"""
+
+from datetime import datetime
+
+from typing import Literal
+from pydantic import BaseModel, Field
+
+
+def parse_date(date_str: str) -> datetime:
+    """Parse datetime from the specified format"""
+
+    return datetime.strptime(date_str, "%d/%m/%Y")
```

Comment:
May not be neccessary, you could just create a constant `DATE_FORMAT = "%d/%m/%Y"` at the top of the file and then always simply call `datetime.strptime(..., DATE_FORMAT)`. It's usually not needed to create a function only to call another one-liner.

Reply:
I've given this more thought and I think you should keep this here. I'd just remove the "from the specified format" part from the docstring as it's not true, you are parsing it from a constant format.

Reply:
I eventually decided to remove the function completely (as you say it's just another one-liner), because the format changes with each csv file, what do you think?